rules/no-duplicate-variable/index.html

```diff
@@ -9,7 +9,7 @@
 
   A variable can be reassigned if necessary -
   there's no good reason to have a duplicate variable declaration.
-optionsDescription: You can specify `"check-parameters"` to check for variables with the same name as a paramter.
+optionsDescription: You can specify `"check-parameters"` to check for variables with the same name as a parameter.
```

Comment:
This one is also just generated from the source file on the master branch. As I already said, this is already fixed. The generated docs are updated during the next release.